hardware/edison-sparkfun-io.md

```diff
@@ -1,10 +1,16 @@
 # Sparkfun Blocks I/O
 
+# Sparkfun 扩展模块 I/O
+
 <p>This section describes the <a href="https://developer.android.google.cn/things/sdk/pio/index.html">Peripheral I/O</a>
```

Comment:
This section describes the [Peripheral I/O](https://developer.android.google.cn/things/sdk/pio/index.html) interfaces available to your apps running on the [Raspberry Pi Block for Intel® Edison](https://www.sparkfun.com/products/13044).